src/base/bittorrent/torrentimpl.h

```diff
@@ -275,8 +275,8 @@ namespace BitTorrent
         void deferredRequestResumeData();
         void handleMoveStorageJobFinished(const Path &path, MoveStorageContext context, bool hasOutstandingJob);
         void fileSearchFinished(const Path &savePath, const PathList &fileNames);
-        TrackerEntry updateTrackerEntry(const lt::announce_entry &announceEntry, const QHash<lt::tcp::endpoint, QMap<int, int>> &updateInfo);
-        void resetTrackerEntries();
+        TrackerEntryInfo updateTrackerEntryInfo(const lt::announce_entry &announceEntry, const QHash<lt::tcp::endpoint, QMap<int, int>> &updateInfo);
+        void resetTrackerEntryInfo();
```

Comment:
Here is an example of a confusing name. It looks like it affects a single object.

Reply:
Any suggestions? `resetAllTrackerEntryInfo`?